tests/libyui/menu_hotkeys_1177760_spec.rb

```diff
@@ -0,0 +1,36 @@
+require_relative "rspec_tmux_tui"
+
```

Comment:
NP: It would be nice to have a full bug URL here, just having a bug number might not be enough for people not familiar with libyui or SUSE.